xds/src/main/java/io/grpc/xds/XdsClientWrapperForServerSds.java

```diff
@@ -90,34 +88,6 @@ public XdsClientWrapperForServerSds(int port) {
     this.port = port;
   }
 
-  private SynchronizationContext createSynchronizationContext() {
-    final InternalLogId logId =
-        InternalLogId.allocate("XdsClientWrapperForServerSds", Integer.toString(port));
```

Comment:
The `InternalLogId` functionality is lost in the new SyncContext in the AbstractXdsClient`. Why can't it be provided?

Reply:
The XdsClient instance has an unique logId.